Classes/Audio/ParallelAudioPlayer.cs

```diff
@@ -46,7 +46,7 @@ public class ParallelAudioPlayer: IDisposable {
                 Volume = defaultVolume
             };
             notePlayers[i] = new WasapiOut(AudioClientShareMode.Shared, desiredLatency);
-            if (isPanned) {
+            if (isPanned && basePath != "mmatick") {
```

Comment:
what's the reason for this change?

Reply:
The WAV they're using in MMA2 only has one channel, so StereoToMonoSampleProvider throws an exception when the note sound panning is enabled.